cmake/AppImageKitConfig.cmake.in

```diff
@@ -1,17 +1,23 @@
 # - Config file for the AppImage package
 # It defines the following variables
-#  APPIMAGE_INCLUDE_DIRS - include directories for APPIMAGE
-#  APPIMAGE_LIBRARIES    - libraries to link against
-#  APPIMAGE_EXECUTABLE   - the bar executable
- 
+#  APPIMAGEKIT_INCLUDE_DIRS - include directories for APPIMAGE
+#  APPIMAGEKIT_LIBRARIES    - libraries to link against
+
 # Compute paths
-get_filename_component(APPIMAGE_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-set(APPIMAGE_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
- 
+get_filename_component(APPIMAGEKIT_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+set(APPIMAGEKIT_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
+
 # Our library dependencies (contains definitions for IMPORTED targets)
-if(NOT TARGET foo AND NOT APPIMAGE_BINARY_DIR)
-  include("${APPIMAGE_CMAKE_DIR}/AppImageTargets.cmake")
-endif()
- 
-# These are IMPORTED targets created by AppImageTargets.cmake
-set(APPIMAGE_LIBRARIES foo)
+include("${APPIMAGEKIT_CMAKE_DIR}/AppImageKitTargets.cmake")
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(GLIB REQUIRED glib-2.0)
+pkg_check_modules(GIO REQUIRED gio-2.0)
+pkg_check_modules(ZLIB REQUIRED zlib)
+pkg_check_modules(CAIRO REQUIRED cairo)
+pkg_check_modules(OPENSSL REQUIRED openssl)
+
+# These are IMPORTED targets created by AppImageKitTargets.cmake
+set(APPIMAGEKIT_LIBRARIES
+  libappimage
```

Comment:
For linking, most likely `-lappimage` is required, hence this value would have to be called `appimage`. However, that's a pretty inferior solution, given that CMake has a nifty standard macro called `find_library`.

```
find_library(LIBAPPIMAGE_PATH
    NAMES appimage
    [PATHS @LIBRARY_PATH_IF_WE_HAVE_SUCH_A_VARIABLE@]
)


set(APPIMAGE_LIBRARIES ${LIBAPPIMAGE_PATH})
```

https://cmake.org/cmake/help/v3.0/command/find_library.html

Usually one doesn't have to specify a library path.

Reply:
This is embarrassing, that data shouldn't be commit in first time. Ctest is Ok  without them.

Reply:
I'm referring here not to the libappimage.so but to the libappimage target, the link libraries are defined in the AppImageKitTargets.cmake file

Reply:
As far as I can see it's Ok this way. Can we just wait for some distribution maintainer or developer to complain about this in order to change it?